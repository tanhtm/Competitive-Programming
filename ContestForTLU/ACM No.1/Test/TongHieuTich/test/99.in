90
667 483
847 308
311 282
593 654
-345 43
-364 -893
-750 751
-947 389
871 -506
-165 621
-903 132
137 -730
-124 500
392 -260
-362 -682
-70 931
-436 -762
735 923
406 778
-859 -967
-819 -7
618 -808
490 870
800 911
24 -564
-838 -669
-446 -228
547 865
-223 456
-264 629
498 -162
-72 -866
429 126
-146 303
-530 -351
-215 234
42 718
698 152
-314 775
-992 136
261 93
395 -252
-602 -299
-884 495
564 585
-88 693
-820 59
194 25
117 973
224 265
-915 490
336 -948
-884 360
-404 375
-786 213
589 -616
-941 -53
318 -706
-547 684
-631 -809
168 349
-76 912
124 -531
498 -328
-380 41
746 942
625 -366
526 660
-360 661
924 113
643 -838
-517 -583
623 531
472 -281
998 -539
-550 232
-892 -535
-647 694
531 28
611 710
240 295
-575 -669
559 -269
500 367
-901 -768
-813 -476
-503 24
-733 -740
-386 -706
-727 600
