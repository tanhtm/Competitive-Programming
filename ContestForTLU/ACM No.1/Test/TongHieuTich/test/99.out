52900
155636
4299
-19016
29294
-166238
-375
186372
125651
-89604
199496
-128532
-58656
21516
-83520
-215465
-97637
-77926
-110112
-49302
167678
-67735
-129200
-47480
-79380
63670
36733
-112254
-39551
-81486
55440
-186193
42041
-17623
39424
-2132
-128440
116025
-125507
241392
14868
23130
68250
134107
-6032
-118126
167229
9252
-233260
-5012
149281
-196452
162964
5647
143106
-8133
220668
-99328
-42161
-64080
-23394
-206492
-66646
35105
35679
-82712
64167
-39731
-76830
210251
-72198
-18150
26542
35955
176370
62169
127359
-15756
70294
-32694
-7356
-29234
60030
28827
55494
108598
63108
-2577
-87360
42132
